### ISSUE TITLE
pypy -m pip install persistent==4.6.0 fails with ERROR: Could not install packages due to an EnvironmentError: [Errno 13] Permission denied: '/home/travis/virtualenv/pypy2.7-7.1.1/include/site'

### ISSUE THREAD

**Issue by mgedmin** [zopefoundation/persistent#135]
I'm now seeing random projects that depend on persistent failing pypy and pypy3 jobs with

```
  ERROR: Command errored out with exit status 1:
   command: /home/travis/virtualenv/pypy2.7-7.1.1/bin/pypy /home/travis/virtualenv/pypy2.7-7.1.1/site-packages/pip install --ignore-installed --no-user --prefix /tmp/pip-build-env-i84YmT/overlay --no-warn-script-location --no-binary :none: --only-binary :none: -i https://pypi.org/simple -- setuptools wheel persistent
       cwd: None
  Complete output (18 lines):
  DEPRECATION: A future version of pip will drop support for Python 2.7. More details about Python 2 support in pip, can be found at https://pip.pypa.io/en/latest/development/release-process/#python-2-support
  Collecting setuptools
    Using cached setuptools-44.0.0-py2.py3-none-any.whl (583 kB)
  Collecting wheel
    Using cached wheel-0.34.2-py2.py3-none-any.whl (26 kB)
  Collecting persistent
    Using cached persistent-4.6.0.tar.gz (119 kB)
  Processing /home/travis/.cache/pip/wheels/cb/36/23/b548e9f4086bd66fe3d50a345973c9c2a9a29d5f30d6e37684/zope.interface-4.7.1-py2-none-any.whl
  Building wheels for collected packages: persistent
    Building wheel for persistent (setup.py): started
    Building wheel for persistent (setup.py): finished with status 'done'
    Created wheel for persistent: filename=persistent-4.6.0-pp271-pypy_41-linux_x86_64.whl size=150148 sha256=9bc0d169395d25b2787c3d627fcdd6739eeeb97d3bc4e63cd427de850376299e
    Stored in directory: /home/travis/.cache/pip/wheels/45/93/6f/93701ce04a6e41acebe3aff780111e7d651da3f826657a081d
  Successfully built persistent
  Installing collected packages: setuptools, wheel, zope.interface, persistent
  ERROR: Could not install packages due to an EnvironmentError: [Errno 13] Permission denied: '/home/travis/virtualenv/pypy2.7-7.1.1/include/site'
  Consider using the `--user` option or check the permissions.
```

Now this might be a pypy bug, or virtualenv bug, or persistent bug, and I'm not sure which, but I'd like to track this here.

**Comment by mgedmin** [comment]
Here's an example:
- successful build: https://travis-ci.org/mgedmin/zodbbrowser/jobs/657834841
- failed cron build of the same commit a day later: https://travis-ci.org/mgedmin/zodbbrowser/jobs/658772940

pypy version is the same; pip version is the same; persistent was 4.5.1 and now is 4.6.0.

**Comment by mgedmin** [comment]
Oh hey in 4.5.1 we had a setup.py that did

    if is_pypy:
        headers = []

and in 4.6.0 it always sets

    headers = [
       'persistent/cPersistence.h',
       'persistent/ring.h',
    ]

so that might explain everything.

**Comment by mgedmin** [comment]
Fixed in 4.6.1.
